woocommerce/wc-admin
# Update analytics card header text styles

This PR updates the analytics card header text to use the `subtitle` styles from the @wordpress/components package. Also minor updates were made to the chart header titles for size color and margin. 

### Accessibility




### Screenshots

**Before**

<img width="1478" alt="Screen Shot 2021-03-03 at 12 55 35 PM" src="https://user-images.githubusercontent.com/4500952/109871835-76de2600-7c20-11eb-8c2a-f75714ed3088.png">

<img width="1478" alt="Screen Shot 2021-03-03 at 12 55 11 PM" src="https://user-images.githubusercontent.com/4500952/109871830-75acf900-7c20-11eb-8058-dcddcf664dfb.png">

<img width="1479" alt="Screen Shot 2021-03-03 at 12 55 22 PM" src="https://user-images.githubusercontent.com/4500952/109871837-780f5300-7c20-11eb-82d4-5fbf16e30536.png">

**After**

<img width="1478" alt="Screen Shot 2021-03-03 at 12 54 18 PM" src="https://user-images.githubusercontent.com/4500952/109871938-95dcb800-7c20-11eb-800c-c1687106621e.png">

<img width="1477" alt="Screen Shot 2021-03-03 at 12 54 36 PM" src="https://user-images.githubusercontent.com/4500952/109871927-92e1c780-7c20-11eb-9f1d-91e5519a86f0.png">

<img width="1478" alt="Screen Shot 2021-03-03 at 12 53 04 PM" src="https://user-images.githubusercontent.com/4500952/109871940-970de500-7c20-11eb-938b-de4f305e5c56.png">


### Detailed test instructions:

-   Run this branch
-   Verify that the analytics card header sizes have been updated, and that the color and sizing are consistent across the experience.
